dist/main.js

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
@Rodrigoplp-work  what's the reason for adding this file? Please let me know.

Reply:
Removed.